### PR TITLE
function code done and testings all passed

### PR DESCRIPTION
**Pull request by CChCheChen**

### CODE REVIEW

File: R/calculate_grade.R [hunk 1]
```diff
@@ -14,4 +14,41 @@
...
+    cwd <- getwd()
+    path <- paste0(cwd, "/", input_file_path)
+    course_info <- read.csv(path, header = TRUE)
+    # course_info <- read.csv(input_file_path, col.names = TRUE, header = FALSE)
+
+    print(course_info)
+
+    error_msg_missing_value <- "Course component grades is missing for "
+    error_msg_format <- "Course component grades with incorrect format (not 2 decimal places) for "
+    grade_msg <- "Course garde is "
```
Comment (shlrley) [inline R/calculate_grade.R:26]: Typo for "grade"

File: R/calculate_grade.R [hunk 2]
```diff
@@ -14,4 +14,41 @@
...
+    percent_sign <- "%"
+    course_grade <- 0
+    return_msg <- ""
+
+    # Check if all course components have garde present
```
Comment (shlrley) [inline R/calculate_grade.R]: Typo for "grade"

Comment (CChCheChen) [reply R/calculate_grade.R]: I will update :(

Comment (CChCheChen) [reply R/calculate_grade.R]: @shlrley the typo is now fixed with a new commit and push here again

**Review by shlrley — APPROVED** [review]

**Pull request merged**